File: math/11050.cpp
#include <iostream>
using namespace std;

int main() {
    int N, K, res = 1;
    cin >> N >> K;
    int i;
    for (i = 0; i < K; i++) {
        res *= N--;
    }
    for (i = 1; i < K + 1; i++) {
        res /= i;
    }

    cout << res << endl;
    return 0;
}

File: math/11050.py
from sys import stdin
input = stdin.readline

def compute(n: int, k: int) -> int:
    res = 1
    for i in range(n, n - k, -1):
        res *= i
    for i in range(1, k + 1):
        res //= i
    return res

if __name__ == '__main__':
    n, k = map(int, input().split())
    print(compute(n, k))
